app/styleguide/page.tsx

```diff
@@ -1037,23 +1059,189 @@ function Page() {
 
                 <div><pre><code>
                   {`.grid-view {
-                    --_gap: var(--grid-list-grid-gap, 1rem);
+    --_gap: var(--grid-list-grid-gap, 1rem);
 
-                    display: grid;
+    display: grid;
 
-                    padding: var(--_gap);
-                    grid-gap: var(--_gap);
-                    grid-template-columns: repeat(4, 1fr);
-                    grid-template-rows: auto;
-                  }
-                `}
-                </code></pre>
-                </div>
+    padding: var(--_gap);
+    grid-gap: var(--_gap);
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;
+  }
+`}
+                </code></pre></div>
               </li>
             </ul>
+          </div>
+
+          <div id="_containers">
+            <h2><code>Layout Container</code> (Base)</h2>
+            <p>The standard <code>{`<LayoutContainer>`}</code> wraps content containers to provide
+            some common container within the layout container.</p>
+            <LayoutContainer>
```

Comment:
Will ContentContainer always live inside "LayoutContainer"? If so, we could update LayoutContainer to always include "ContentContainer".

Reply:
I did consider this, but decided against it because there is always a possibility that we might have a `LayoutContainer` that have multiple `ContentContainer`.

We could for example do something like this:
```html
<LayoutContainer>
  <ToolbarContainer>...</ToolbarContainer>
  <ContentContainer className="profile-form"> ... </ContentContainer>
  <ContentContainer className="contributors-form">... </ContentContainer>
</LayoutContainer>
```